README.md

```diff
@@ -50,9 +50,11 @@ For a full CI run
 
 ## Release notes
 
-### 3.2.0 (dev)
+### 3.2.0 (2017-05-09)
```

Comment:
> Is composer clever enough to balk when a PHP 5.6 project has a ~3.1 dependency on FileFetcher?

Yes it is smart enough. It will not pull in code requiring a higher PHP version (or version of another dependency) that it cannot accommodate.

> I'd consider the changes you made in the interface a BC break, so wouldn't it be better to call this version 4.0?

Yes, thanks for pointing this out. I also realized this when walking home yesterday and am happy to be reminded now. Someone running the Maps extension with PHP7 would get test failures due to an implementation of the interface there that does not have the type hints. (Same for other users with an implementation ofc.)